CMakeLists.txt

```diff
@@ -45,45 +50,10 @@ execute_process(COMMAND "${CYTHON_EXECUTABLE}" "-V"
     ERROR_STRIP_TRAILING_WHITESPACE)
 message( STATUS "Cython Version: " ${CYTHON_VERSION} )
 
-# Find LibXML++ and dependencies
-FIND_PACKAGE(LibXML++ REQUIRED)
-SET(CYMETRIC_INCLUDE_DIR ${CYMETRIC_INCLUDE_DIR} ${LibXML++_INCLUDE_DIR} 
-    ${Glibmm_INCLUDE_DIRS} ${LibXML++Config_INCLUDE_DIR})
-SET(LIBS ${LIBS} ${LibXML++_LIBRARIES})
-
-# Include the boost header files, system, and filesystem libraries
-SET(Boost_USE_STATIC_LIBS       OFF)
-SET(Boost_USE_STATIC_RUNTIME    OFF)
-FIND_PACKAGE(Boost COMPONENTS filesystem system REQUIRED)
-SET(CYMETRIC_INCLUDE_DIR ${CYMETRIC_INCLUDE_DIR} ${Boost_INCLUDE_DIR})
-SET(LIBS ${LIBS} ${Boost_FILESYSTEM_LIBRARY})
-SET(LIBS ${LIBS} ${Boost_SYSTEM_LIBRARY})
-
-# find lapack and link to it
-FIND_PACKAGE(LAPACK REQUIRED)
-set(LIBS ${LIBS} ${LAPACK_LIBRARIES})
-MESSAGE("\tFound LAPACK Libraries: ${LAPACK_LIBRARIES}")
-
-# Find HDF5
-FIND_PACKAGE(HDF5 REQUIRED)
-ADD_DEFINITIONS(${HDF5_DEFINITIONS})
-set(LIBS ${LIBS} ${HDF5_LIBRARIES})
-SET(CYMETRIC_INCLUDE_DIR "${HDF5_INCLUDE_DIRS}" ${CYMETRIC_INCLUDE_DIR})
-
```

Comment:
I think all these are still required - we just needed to move the cyclus find to the top.